Makefile

```diff
@@ -30,3 +30,47 @@ shellcheck-gha:
 generate-gha-workflows:
 	./.github/scripts/generate_linux_ci_workflows.py
 	$(MAKE) shellcheck-gha
+
+setup_lint:
+	python tools/actions_local_runner.py --file .github/workflows/lint.yml \
+	 	--job 'flake8-py3' --step 'Install dependencies'
+	python tools/actions_local_runner.py --file .github/workflows/lint.yml \
+	 	--job 'cmakelint' --step 'Install dependencies'
+	pip install jinja2
+
+quick_checks:
+# TODO: This is broken when 'git config submodule.recurse' is 'true'
+	@python tools/actions_local_runner.py \
```

Comment:
I don't think it's actually necessary to hide the command being executed here, if anything I think it's more valuable to the user to know exactly what command is being executed in the makefile

Reply:
Any extra output makes it harder to parse for errors, especially since all these tools have disparate ways of reporting their results, here I was thinking that for a successful run there should be no output at all, whereas outputting all the makefile commands and the scripts creates a lot of spew:

```
./.github/scripts/generate_linux_ci_workflows.py
make: ./.github/scripts/generate_linux_ci_workflows.py: Command not found
make: *** [Makefile:23: generate-gha-workflows] Error 127
make: *** Waiting for unfinished jobs....
quick-checks: Ensure no unqualified noqa
quick-checks: Ensure no direct cub include
quick-checks: Ensure canonical include
quick-checks: Ensure no tabs
quick-checks: Ensure no non-breaking spaces
quick-checks: Ensure correct trailing newlines
cmakelint: Run cmakelint
quick-checks: Ensure no trailing spaces
mypy: Run mypy
Success: no issues found in 1316 source files
Success: no issues found in 56 source files
flake8-py3: Run flake8
```

vs

```
python tools/actions_local_runner.py \
        --file .github/workflows/lint.yml \
        --job 'flake8-py3' \
        --step 'Run flake8'
python tools/actions_local_runner.py \
        --file .github/workflows/lint.yml \
        --job 'mypy' \
        --step 'Run mypy'
python tools/actions_local_runner.py \
        --file .github/workflows/lint.yml \
        --job 'quick-checks' \
        --step 'Ensure no trailing spaces' \
        --step 'Ensure no tabs' \
        --step 'Ensure no non-breaking spaces' \
        --step 'Ensure canonical include' \
        --step 'Ensure no unqualified noqa' \
        --step 'Ensure no direct cub include' \
        --step 'Ensure correct trailing newlines'
python tools/actions_local_runner.py \
        --file .github/workflows/lint.yml \
        --job 'cmakelint' \
        --step 'Run cmakelint'
./.github/scripts/generate_linux_ci_workflows.py
make: ./.github/scripts/generate_linux_ci_workflows.py: Command not found
make: *** [Makefile:23: generate-gha-workflows] Error 127
make: *** Waiting for unfinished jobs....
quick-checks: Ensure no unqualified noqa
quick-checks: Ensure no direct cub include
quick-checks: Ensure canonical include
quick-checks: Ensure no non-breaking spaces
quick-checks: Ensure no tabs
quick-checks: Ensure correct trailing newlines
cmakelint: Run cmakelint
+ git ls-files -z -- bootstrap '*.cmake' '*.cmake.in' '*CMakeLists.txt'
+ grep -E -z -v '^(cmake/Modules/|cmake/Modules_CUDA_fix/|cmake/Caffe2Config.cmake.in|aten/src/ATen/ATenConfig.cmake.in|cmake/Caffe2ConfigVersion.cmake.in|cmake/TorchConfig.cmake.in|cmake/TorchConfigVersion.cmake.in|cmake/cmake_uninstall.cmake.in)'
+ xargs -0 cmakelint --config=.cmakelintrc --spaces=2 --quiet
quick-checks: Ensure no trailing spaces
mypy: Run mypy
+ for CONFIG in mypy*.ini
+ mypy --config=mypy.ini
+ for CONFIG in mypy*.ini
+ mypy --config=mypy-strict.ini
Success: no issues found in 1316 source files
Success: no issues found in 56 source files
flake8-py3: Run flake8
+ flake8
+ tee /tmp/flake8-output.txt
```

Reply:
It's also possible to force the commands to be shown (at least on the Makefile), though the ergonomics aren't great

```bash
make lint SHELL='sh -x'
```